beeline.go

```diff
@@ -52,7 +53,20 @@ type Config struct {
 	// event before it gets sent to Honeycomb. Does not get invoked if the event
 	// is going to be dropped because of sampling. Runs after the SamplerHook.
 	PresendHook func(map[string]interface{})
-
+	// TraceHeaderParserHook is a function call that will get run with the context
+	// and HTTP request on all incoming requests. The function registered may
+	// construct and return a SpanContext, which will be used in the subsequent
+	// creation of trace and span objects. This is useful when parsing trace
+	// propagation headers from upstream services. HTTPPropagator objects defined
+	// in the propagation package or elsewhere may be used to do the parsing.
+	TraceHTTPHeaderParserHook func(ctx context.Context, r *http.Request) (*trace.SpanContext, context.Context)
+	// TraceHTTPHeaderPropagationHook is a function call that will get run with
```

Comment:
```suggestion
	// TraceHTTPHeaderPropagationHook is a function that will be invoked with
```

Reply:
I like this suggestion - changed throughout (there were a few other cases of "get run with").